packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/WithdrawalMethods/selectors.ts

```diff
@@ -35,7 +35,9 @@ export const getData = state => {
       paymentMethods:
         (!invitations.openBanking && {
           ...paymentMethods,
-          methods: paymentMethods.methods.filter(m => m.type === 'BANK_ACCOUNT')
+          methods: paymentMethods.methods.filter(m => {
+            return m.type === 'BANK_ACCOUNT' || m.currency === 'USD'
+          })
```

Comment:
should we implicitly return?

Reply:
Should be fine for now. This code will disappear once it's rolled out to everyone.